### PR TITLE
Organization users pagination

### PR DESCRIPTION
**Pull request by jayendrac**
#### What does this PR do?
Master data default page size is 15 so all the users in organization are not displayed.

Issue Fixed: When the organization has more than 15 users and someone edited their profile, natural order of master data entries gets changed so it throws an error finding default organization.

Feature: Implement pagination for users in organization

#### How to test it? 
You can see the pagination in the screenshot
image

you can check the [demo](https://demo--b2bstore.myvtex.com/account#/users) workspace in b2bstore by sign in as "Fernando"

### CODE REVIEW

File: react/components/MyUsers.tsx [hunk 1]
```diff
@@ -247,6 +271,16 @@ const MyUsers = ({
...
           </div>
         </div>
+        <div className="flex justify-center">
+          {
+            loadingAssignments || assignments.length >= assignmentsPageSize ? 
+            <Button 
+            size="small" 
+            onClick={loadMoreAssignments}
+            isLoading={loadingAssignments}
+            >Show More</Button> : <div />
```
Comment (lucis) [inline react/components/MyUsers.tsx:281]: Localize this `"Show More"`

File: react/components/MyUsers.tsx [hunk 2]
```diff
@@ -43,6 +43,9 @@ const MyUsers = ({
   showToast,
   intl,
 }: Props) => {
+  const PAGE_SIZE_STEPPER = 10
+  const [assignmentsPageSize, setAssignmentsPageSize] = useState(10)
```
Comment (lucis) [inline react/components/MyUsers.tsx:47]:
```suggestion
  const [assignmentsPageSize, setAssignmentsPageSize] = useState(PAGE_SIZE_STEPPER)
```

File: react/components/MyUsers.tsx (hunk 1 above)
Comment (Jayendra88) [reply react/components/MyUsers.tsx]: Fixed

File: react/components/MyUsers.tsx (hunk 2 above)
Comment (Jayendra88) [reply react/components/MyUsers.tsx]: Fixed

**Pull request merged**